File: javascript/src/models/telnyxConfigOut.ts
// this file is @generated

export interface TelnyxConfigOut {
  publicKey: string;
}

export const TelnyxConfigOutSerializer = {
  _fromJsonObject(object: any): TelnyxConfigOut {
    return {
      publicKey: object["publicKey"],
    };
  },

  _toJsonObject(self: TelnyxConfigOut): any {
    return {
      publicKey: self.publicKey,
    };
  },
};

File: rust/src/models/telnyx_config_out.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize)]
pub struct TelnyxConfigOut {
    #[serde(rename = "publicKey")]
    pub public_key: String,
}

impl TelnyxConfigOut {
    pub fn new(public_key: String) -> Self {
        Self { public_key }
    }
}
